### ISSUE TITLE
[Perf] Ubuntu16.04 runs blocked by multiple "PE file is already strong-name signed." errors

### ISSUE THREAD

**Issue by jorive** [dotnet/corefx#29331]
[perf_ubuntu16.04_release](https://ci2.dot.net/job/dotnet_corefx/job/perf/job/master/job/perf_ubuntu16.04_release/1715/consoleText)

```bash
bash ./build-managed.sh -release -tests -- /p:Performance=true /p:TargetOS=Linux /m:1 /p:LogToBenchview=true /p:BenchviewRunType=rolling /p:PerformanceType=Profile
```
...

/home/administrator/jenkins/w/dotnet_corefx/perf/master/perf_ubuntu16.04_release/Tools/sign.targets(113,5): error : /home/administrator/jenkins/w/dotnet_corefx/perf/master/perf_ubuntu16.04_release/bin/obj/ref/System.Threading.Tasks.Dataflow/4.6.3.0/netstandard/System.Threading.Tasks.Dataflow.dll: PE file is already strong-name signed. [/home/administrator/jenkins/w/dotnet_corefx/perf/master/perf_ubuntu16.04_release/src/System.Threading.Tasks.Dataflow/ref/System.Threading.Tasks.Dataflow.csproj]
    0 Warning(s)
    32 Error(s)

**Comment by jorive** [comment]
First failure occurred on `Apr 24, 2018 10:19 AM`

**Comment by stephentoub** [comment]
I hit this every once in a while.  I delete the the target .dll it's complaining about, and then it continues on happily.

**Comment by jorive** [comment]
This is happening in the lab when running `build-managed.sh`. Does the script have a retry switch?

**Comment by pjanotti** [comment]
I've never hit this before but started yesterday... also Ubuntu 16.04.

**Comment by pjanotti** [comment]
Something seems to have changed and also repros on macOS. If I do the following:
```
./clean.sh
./build.sh
./build.sh
```
The second build fails complaining about all (?) dlls under bin/obj/ref/**/netstandard/*.dll - this didn't happen before.

1.

**Comment by hughbe** [comment]
I can reproduce this and its really annoying for contributions:

```sh
git clone https://github.com/hughbe/corefx.git
cd corefx
./build.sh
cd src/System.Drawing.Common/tests
../.../../tools/msbuild.sh /T:BuildAndTest
```

```
/Users/hugh/Documents/GitHub/corefx/Tools/sign.targets(113,5): error : /Users/hugh/Documents/GitHub/corefx/bin/obj/AnyOS.AnyCPU.Debug/RemoteExecutorConsoleApp/netstandard/RemoteExecutorConsoleApp.exe: PE file is already strong-name signed. [/Users/hugh/Documents/GitHub/corefx/src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.csproj]
```

**Comment by adiaaida** [comment]
This is happening in the release/2.2 perf runs again on Ubuntu 16.04.

@dotnet/corefx-contrib does anyone know why this might be happening now?

**Comment by joshfree** [comment]
@weshaggard @dagood PTAL

**Comment by danmosemsft** [comment]
@weshaggard @eerhardt how can I rebuild and gather more detailed logging? 

There is no indication in the job what is signing it the first time.

If I do rebuild I don't see a space for eg build parameters (like `-- /flp:v=diag` or something)
https://ci2.dot.net/job/dotnet_corefx/job/perf/job/release_2.2/job/perf_ubuntu16.04_release/lastCompletedBuild/rebuild/parameterized

I probably don't want this logging going to the console, but to a file I can get to.

**Comment by eerhardt** [comment]
I'm not sure. @weshaggard ?

**Comment by dagood** [comment]
> how can I rebuild and gather more detailed logging?

@mmitche I think would be right to help with this.

**Comment by mmitche** [comment]
It doesn't look like the jobs are set up to take extra parameters. So...you could push a new branch that adds logging, then run the job against that branch.  The job is currently set up to archive msbuild.log

**Comment by danmosemsft** [comment]
@adamsitnik could you please help get better logs so we can figure this out?

**Comment by brianrob** [comment]
@adamsitnik were you able to make any progress on this?  I'd like to get some results for Linux 2.2.  Thanks.

**Comment by wtgodbe** [comment]
@adamsitnik any update here?

**Comment by ViktorHofer** [comment]
Is this still happening @adiaaida ?

**Comment by adiaaida** [comment]
@ViktorHofer the release/2.2 job has not run since December 3, when, yes, it was still failing. Running now:

https://ci2.dot.net/job/dotnet_corefx/job/perf/job/release_2.2/job/perf_ubuntu16.04_release/61/

**Comment by ViktorHofer** [comment]
Same error is still happening. cc @adamsitnik

**Comment by ericstj** [comment]
@ViktorHofer does this repro in 3.0?  If not then I imagine we won't fix in the 3.0 milestone.

**Comment by ViktorHofer** [comment]
This can be closed, we don't have perf jobs anymore in corefx.
